test/e2e-common.sh

```diff
@@ -39,7 +39,26 @@ function knative_setup() {
   echo ">> Starting Knative Eventing"
   echo "Installing Knative Eventing"
   ko apply -f ${EVENTING_CONFIG} || return 1
+
+  # Force tracing to 100%.
+  kubectl apply -f - << END
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: knative-eventing
+  name: config-tracing
+data:
+  backend: stackdriver
+  zipkin-endpoint: "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans"
+  debug: "true"
+  sample-rate: "1.0"
+END
   wait_until_pods_running knative-eventing || fail_test "Knative Eventing did not come up"
+
+  echo "Installing Knative Monitoring"
+  kubectl create namespace istio-system
+  kubectl apply --filename https://github.com/knative/serving/releases/download/v0.8.0/monitoring-tracing-zipkin-in-mem.yaml || return 1
```

Comment:
It seems we should always install the latest release version. https://github.com/knative/test-infra/pull/1354 exposes the release URL for `monitoring`, after it's merged, we will be install it with `kubectl apply -f ${KNATIVE_MONITORING_RELEASE}`

Reply:
Done.